kedro/framework/project/__init__.py

```diff
@@ -179,8 +187,26 @@ def configure(self, pipelines_module: Optional[str] = None) -> None:
     __str__ = _load_data_wrapper(str)
 
 
+class _ProjectLogging(UserDict):
+    def __init__(self):
+        default_logging = (Path(__file__).parent / "default_logging.yml").read_text()
+        self.configure(yaml.safe_load(default_logging))
+
+        # We suppress click here to hide tracebacks related to it conversely,
+        # kedro is not suppressed to show its tracebacks for easier debugging.
+        # sys.executable is used to get the kedro executable path to hide the top level traceback.
+        rich_traceback_install(
+            show_locals=True, suppress=[click, str(Path(sys.executable).parent)]
+        )
+        logging.captureWarnings(True)
+
+    def configure(self, logging_config: Dict[str, Any]) -> None:
+        logging.config.dictConfig(logging_config)
+        self.data = logging_config
+
+
 PACKAGE_NAME = None
-LOGGING = None
+LOGGING = _ProjectLogging()
```

Comment:
The major success here is that this now acts like `pipelines` and `settings`, and `configure_logging` is much closer to `configure_project`.